log4j-api-test/src/test/java/org/apache/logging/log4j/message/ParameterizedMessageTest.java

```diff
@@ -115,7 +115,7 @@ public void testFormatStringArgsWithTrailingEscapedEscape() {
         final String testMsg = "Test message {}{} {}\\\\";
         final String[] args = { "a", "b", "c" };
         final String result = ParameterizedMessage.format(testMsg, args);
-        assertEquals("Test message ab c\\\\", result);
+        assertEquals("Test message ab c\\", result);
```

Comment:
@rgoers, this is changed due to the same reason I have elaborated on in `ParameterFormatterTest` comments.

Reply:
It has been two weeks. I am resolving this conversation.